samples/dotnet_core_managed_gateway/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "dotnet_core_managed_gateway": {
+      "commandName": "Project",
+      "commandLineArgs": "C:\\repos\\azure-iot-gateway-sdk\\samples\\dotnet_core_managed_gateway\\dotnet_core_managed_gateway_win.json"
```

Comment:
This file shall be removed. It was not supposed to be checked in. 

---
In reply to: [105794179](https://github.com/Azure/azure-iot-gateway-sdk/pull/163#discussion_r105794179) [](ancestors = 105794179)